<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regulation_groups_groupbox</class>
 <widget class="QGroupBox" name="regulation_groups_groupbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="libraries_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Kaavamääräysryhmäkirjastot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="plan_regulation_group_libraries_combobox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="regulation_groups_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="regulation_groups_label">
        <property name="text">
         <string>Kaavakohteen kaavamääräysryhmät</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="plan_regulation_group_scrollarea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="plan_regulation_group_scrollarea_contents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>599</width>
           <height>511</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
